src/components/LibraryItem.tsx

```diff
@@ -248,18 +251,20 @@ export class LibraryItem extends React.Component<LibraryItemProps, LibraryItemSt
         }
     }
 
-    onLibraryItemHoveredOn() {
+    onLibraryItemMouseLeave() {
         let libraryContainer = this.props.libraryContainer;
         if (this.props.data.childItems.length == 0) {
-            libraryContainer.raiseEvent(libraryContainer.props.libraryController.ItemHoveredOnEventName,
-                this.props.data.contextData);
+            libraryContainer.raiseEvent(libraryContainer.props.libraryController.ItemMouseLeaveEventName, true);
```

Comment:
I think it will make sense to send the context data with MouseLeave event as well so that client won't have to cache and it can figure out which item mouse has left.